### PR TITLE
Remap Penn museum

### PR DESCRIPTION
**Pull request by jacobthill**
## Why was this change made?

- revise penn config

## Was the documentation (README, API, wiki, ...) updated?
 n/a

### CODE REVIEW

**Comment by jacobthill** [comment]
@mjgiarlo, @aaron-collier this is probably super easy but how can I assign values to the accumulator in the penn.rb file in this PR? I want to replace 'Manuscript Fragment' with 'Manuscript', leave 'Manuscript' values as is, and replace anything else with 'Cultural Artifact' but I can't figure out how to overwrite the value. In python its something like: 

```
if accumulator == 'Manuscript Fragment':
    accumulator = 'Manuscript'
```
How do I do the equivalent in ruby?

File: lib/macros/penn.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    #   penn_cho_has_type => lambda { ... }
+    # @return [Proc] a proc that traject can call for each record
+    def penn_cho_has_type
+      lambda do |_record, accumulator|
+        if accumulator.first == 'Manuscript Fragment'
+          accumulator.first.gsub('Manuscript Fragment', 'MS WORKED')
+        elsif accumulator.first == 'Manuscript'
+          accumulator.first.gsub('Manuscript', 'Manuscript ALSO WORKED')
+        else
+          accumulator.first.gsub(/./, 'Cultural Artifact')
```
Comment (mjgiarlo) [inline lib/macros/penn.rb]: @jacobthill is this the line that's not doing what you expect? Give `accumulator.replace(['Cultural Artifact'])` a try.

Comment (jacobthill) [reply lib/macros/penn.rb]: Thanks @mjgiarlo that works except I can't get the if condition to match so it replaces everything with ['Cultural Artifact']. How do I check to see if the accumulator is ['Manuscript'] etc. before overwriting it? I've tried
```
if accumulator[0]&.match(['Manuscript Frangment'])
```
and 
```
if accumulator[0]&.match('Manuscript Frangment')
```
and
```
if accumulator.first&.match('Manuscript Frangment')
```
etc.

Comment (jacobthill) [reply lib/macros/penn.rb]: Nevermind, I got it. thanks

**Pull request merged**